lib/ffprobe.ex

```diff
@@ -41,15 +42,24 @@ defmodule FFprobe do
   Get the "format" map, containing general info for the specified file,
   such as number of streams, duration, file size, and more.
   """
-  @spec format(binary) :: format_map
+  @spec format(binary) :: format_map | no_return
```

Comment:
These always return a value, even if it might be an empty map/list, so please remove `no_return`

Reply:
You're right, thanks.